.gitignore

```diff
@@ -1,5 +1,4 @@
 /sequins
-/sequins-dump
```

Comment:
It was purely a tool to dump sequencefiles; I intend to add a similar tool to the sequencefile library at some point. But it's actually not really useful for sequins, anyway; it used to be that the on-disk storage format was sequencefiles, so you could use it to dump out what sequins had locally. Now that's not the case.